### PR TITLE
Vet Clinic Database: create animals table

### PR DESCRIPTION
**Pull request by lugard1**
# Description
In this project I:
- [x] Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate my repo.
- [x] Create a table named animals.
- [x] Copy the SQL I used in the previous point into the schema.sql file in the generated repository.
- [x] Insert data into the created table.
- [x] Copy the SQL I used in the previous point into the data.sql file in the generated repository.
- [x] Write queries.
- [x] Take a screenshot of the results of my queries.
- [x] Copy the SQL I used in the previous point into the queries.sql file in the generated repository.
- [x] Add the screenshot with the results of my queries to the Pull Request description.
- [x] Use Git-flow.
result2 for 1 2
result2 for 3 4
result2 for 5 6
result2 for 7 8

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (1,'Agumon','2020-02-03',0,true,10.23);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (2,'Gabumon','2018-11-15',2,true,8);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (3,'Pikachu','2022-1-7',1,false,15.04);
```
Comment (Ismailco) [inline data.sql]: - [ ] Kindly make sure the data you enter here is correct, the `date_of_birth` of the animal is 3 is `2021/01/07` not 2022/01/07`

File: README.md [hunk 2]
```diff
@@ -102,19 +102,20 @@ After you're finished please remove all the comments and instructions!
...
+- **[Queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-
+<!-- 
 > Add a link to your deployed project.
```
Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] I suggest you remove this section all together because you're not using a live demo

File: README.md [hunk 3]
```diff
@@ -200,37 +211,37 @@ Example:
...
 
-> Describe 1 - 3 features you will add to the project.
+<!-- > Describe 1 - 3 features you will add to the project. -->
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- **I will Query and update animals table**
+<!-- - [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]** -->
```
Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] I suggest you include more features you can include in this project you can think of any feature that would be great to add here

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @lugard1, 

Good job so far 👍🏼 
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (lugard1) [reply data.sql]: Thanks for the insight

File: README.md (hunk 2 above)
Comment (lugard1) [reply README.md]: Thanks for direction.

File: README.md (hunk 3 above)
Comment (lugard1) [reply README.md]: Thanks.

**Review by BenMukebo — APPROVED** [review]
### Hi @lugard1, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly remove this text comment 👇🏻 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**